src/doc/gui_manual/Quantitative/LineSampler.rst

```diff
@@ -0,0 +1,331 @@
+.. _LineSampler:
+
+LineSampler
+-----------
+
+Similar to the Lineout tool one-dimensional curves, created using data
+from 2D or 3D plots, are popular for analyzing data because they are
+simple to compare. However, unlike the Lineout which creates a single
+curve it is a tool, whereas the LineSampler is an operator and can
+create multiple curves.
+
+As an operator the user can define the sampling. That is the user can
+define a series of "arrays" (e.g. planes) that consists of one or more
+"channels" for sampling. For each array the orientation of the plane
+can be defined. Whereas for each channel its orirentation within the
+plane and the sampling type and spacing can be defined. For instance,
+the sampling can be a series of lines through the data, or a series of
+points integrated over time.
+
+Once defined, the operator will produce the appropriate geometry and
+sampling regardless of the window dimension. That is for a 3D window
+the each channel will be drawn showing its 3D position. Similarly for
+a 2D window. For 1D a window the results will be displayed as curve
+plot in much the same way a Lineout data would be drawn.
+
+Unique and critical to using the LineSampler operator is that it
+should be defined in each window for each plot. Further the "Apply to
+all windows" and "Apply operators to all plots" should be checked. As
+such there is some critical setup that must be done.
+
+The genesis of the Line Sampler operator is from plasma physics and
+the synthethic diagnostics performed in fusion simulations so to
```

Comment:
This was already stated in the LineSampler operator section. Does it need to be repeated? If so, perhaps it is best to have it only here?

Remove 'so' before 'to'.